app/services/support_interface/qualifications_export.rb

```diff
@@ -0,0 +1,144 @@
+module SupportInterface
+  class QualificationsExport
+    def data_for_export
+      application_choices = ApplicationChoice
+                                .select(:id, :application_form_id, :rejection_reason, :status, :course_option_id)
+                                .includes(:course_option, :course, :provider)
+
+      application_choices.map do |application_choice|
+        application_form = application_choice.application_form
+        course = application_choice.course_option.course
+        qualifications = application_form.application_qualifications
+        a_levels = a_levels(qualifications).sort_by(&:subject)
+        degrees = degrees(qualifications).sort_by(&:subject)
+
+        output = {
+          'Candidate id' => application_form.candidate_id,
+          'Support ref' => application_form.support_reference,
+          'Phase' => application_form.phase,
+          'Cycle' => application_form.recruitment_cycle_year,
+
+          'Outcome (offer, rejected etc.)' => application_choice.status,
+          'Reason for Rejection' => application_choice.rejection_reason,
```

Comment:
We need to think about adding structured rejection reasons in here too.

Maybe something like `application_choice.structured_rejection_reasons || application_choice.rejection_reason`

Reply:
Nice idea, I've added that in now